### ISSUE TITLE
Uncaught Cannot run python command: check that python is available in the system: Error: spawn python ENOENT

### ISSUE THREAD

**Issue by kuzyr** [sadovnychyi/autocomplete-python#3]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.196.0
**System**: Unknown Windows Version
**Thrown From**: [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python) package, v0.3.0

### Stack Trace

Uncaught Cannot run python command: check that python is available in the system: Error: spawn python ENOENT

```
At C:\Users\dmit1_000\.atom\packages\autocomplete-python\lib\provider.coffee:35

undefined
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ],
    "autoHideMenuBar": true,
    "disabledPackages": [
      "highlight-line"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-python, v0.3.0
fonts, v0.4.2
linter, v0.12.1
linter-python-pep8, v0.2.0
python-isort, v0.0.6
seti-syntax, v0.4.0
seti-ui, v0.6.3

# Dev
No dev packages
```

**Comment by sadovnychyi** [comment]
Can you run `python` from your command line? If no, please check readme about how to make it work on windows.

**Comment by kuzyr** [comment]
Yes, I can. I reinstalled python five minutes ago and python intepretator, python-programmes works.

**Comment by sadovnychyi** [comment]
Hmm, OK. I'll add some default paths to check for python interpreter.

You have it in the c:\python27 or something like this right?

Can you give me an output of `echo %PATH%` please?

**Comment by kuzyr** [comment]
Yes. I have. Did you require it?
![default](https://cloud.githubusercontent.com/assets/12123367/7427068/d250a2a4-efec-11e4-8b8a-c961c1a0978d.png)

**Comment by LorenzLeitner** [comment]
Same here, I have it on E:\Python.

**Comment by sadovnychyi** [comment]
Thats pretty strange, I did tested it on windows 10 and it worked fine **if** python exists in the PATH. Anyway I could add a configuration option to set the python path manually.

**Comment by croscato** [comment]
I was having the same problem. I could made it work putting the python path after the atom editor path. Hope it help.

**Comment by sadovnychyi** [comment]
@croscato just tried it with both: `C:\Python27;C:\Users\sadovnychyi\AppData\Local\atom\bin` and `C:\Users\sadovnychyi\AppData\Local\atom\bin;C:\Python27` -- worked it both cases (Windows 10). But glad that it is resolved for you.

**Comment by sadovnychyi** [comment]
I just released v0.4.0 with better path handling: in addition to system path plugin will try to locate python in some possible locations:
```
'C:\\Python2.7',
'C:\\Python3.4',
'C:\\Python3.5',
'C:\\Program Files (x86)\\Python 2.7',
'C:\\Program Files (x86)\\Python 3.4',
'C:\\Program Files (x86)\\Python 3.5',
'C:\\Program Files (x64)\\Python 2.7',
'C:\\Program Files (x64)\\Python 3.4',
'C:\\Program Files (x64)\\Python 3.5',
'C:\\Program Files\\Python 2.7',
'C:\\Program Files\\Python 3.4',
'C:\\Program Files\\Python 3.5',
```

Also, if its still does not works for you -- you can set it manually in the package settings:
![image](https://cloud.githubusercontent.com/assets/193864/7719539/71167ea2-fef4-11e4-953b-ad37cf860252.png)

I'm closing this issue as resolved, but if you are still experiencing any problems -- feel free to submit a new one.
